### ISSUE TITLE
bias_variance_decomp for keras sequential?

### ISSUE THREAD

**Issue by hanzigs** [rasbt/mlxtend#719]
hi
I have the keras sequential model
```
Keras_fit = Sequential()
Keras_fit.add….
Keras_fit.compile(optimizer = optimizer, loss = 'binary_crossentropy', metrics = ['accuracy'])
```
As keras_fit doesn't have the model, If I only pass the keras_fit to
```
mse, bias, var = bias_variance_decomp(Keras_fit, X_train, y_train, X_valid, y_valid, loss='mse', num_rounds=200, random_seed=1)
```
this will throw
```
AttributeError: 'History' object has no attribute 'predict'
```
May I know how to work this, keras model will be
```
Keras_model = Keras_fit.fit(X_train, y_train, validation_data=(X_valid, y_valid), batch_size = batch_size, epochs = epoch, verbose = 2)
```
Thanks

**Comment by rasbt** [comment]
Hi there,

for your keras model, does it have a predict method? If yes, would the following work?

```python
predictions = Keras_fit.fit(X_boot, y_boot).predict(X_test)
```

**Comment by hanzigs** [comment]
Hi,
It has predict function,
After compiling Keras_fit returns only history not the model, so the Keras_fit.fit(X_boot, y_boot).predict(X_test) throws error like
```
    mse, bias, var = bias_variance_decomp(Keras_fit, X_train, y_train, X_valid, y_valid, loss='mse', num_rounds=200, random_seed=1)
5854/5854 [==============================] - 7s 1ms/step - loss: 0.1229 - accuracy: 0.9544
Traceback (most recent call last):

  File "<ipython-input-21-167ca32306ca>", line 1, in <module>
    mse, bias, var = bias_variance_decomp(Keras_fit, X_train, y_train, X_valid, y_valid, loss='mse', num_rounds=200, random_seed=1)

  File "C:\Users\Anaconda3\lib\site-packages\mlxtend\evaluate\bias_variance_decomp.py", line 77, in bias_variance_decomp
    pred = estimator.fit(X_boot, y_boot).predict(X_test)

AttributeError: 'History' object has no attribute 'predict'
```
Thanks

**Comment by rasbt** [comment]
Hm, I see. I think you would need a wrapper class in this case. The wrapper class would basically implement a `fit()` and a `predict` method, and fit would return a model, not a history. 

I think this could be a super small wrapper class with only a few lines of code. The point is just to override the `fit` function to return a model, not a history.

**Comment by hanzigs** [comment]
Thanks @rasbt,
I have created a pull request for supporting keras Sequential model, It worked, have done testing in my local.
Thanks

**Comment by rasbt** [comment]
Hi there. The fix (thanks @hanzigs !) was merged via #725 and is currently in the development version. You can install the latest development version via

    pip install git+git://github.com/rasbt/mlxtend.git

The code for this fix can be found here: https://github.com/rasbt/mlxtend/pull/725/files

**Comment by phanichintalapati** [comment]
thank you
